01_Data/src/layers/sequelize/model/TransactionEvent/Transaction.ts

```diff
@@ -41,17 +42,17 @@ export class Transaction extends Model implements OCPP2_0_1.TransactionType {
   declare isActive: boolean;
 
   @HasMany(() => TransactionEvent, { as: Transaction.TRANSACTION_EVENTS_ALIAS, foreignKey: 'transactionDatabaseId' })
-  declare transactionEvents?: OCPP2_0_1.TransactionEventRequest[];
+  declare transactionEvents?: object[];
 
   // required only for filtering, should not be used to pull transaction events
   @HasMany(() => TransactionEvent, { as: Transaction.TRANSACTION_EVENTS_FILTER_ALIAS, foreignKey: 'transactionDatabaseId' })
-  declare transactionEventsFilter?: OCPP2_0_1.TransactionEventRequest[];
+  declare transactionEventsFilter?: object[];
 
   @HasMany(() => MeterValue)
-  declare meterValues?: OCPP2_0_1.MeterValueType[];
+  declare meterValues?: MeterValueMapper[];
```

Comment:
After removing `implements OCPP2_0_1.TransactionType`, we can use `MeterValue[]` here.